codechallenges/migrations/0001_initial.py

```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name="CodeChallengeEvent",
```

Comment:
Please don't modify existing migrations, instead please add a new migration that renames the table. We need to do this because we already have a table and data inside this table in production.